app/views/coins/_numismatic_citation_fields.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class='nested-fields'>
```

Comment:
Can we put this in views/shared instead of replicating it for each type that has citations?